python/lsst/obs/hsc/hscMapper.py

```diff
@@ -84,15 +81,9 @@ def __init__(self, **kwargs):
 
         self.addFilters()
 
-        #
-        # self.filters is used elsewhere, and for now we'll set it
-        #
-        # It's a bit hard to initialise self.filters properly until #2113 is resolved,
```

Comment:
Do we know whether this comment is still relevant?

Reply:
#2113 is from trac: https://dev.lsstcorp.org/trac/ticket/2113

We now have `getCanonicalName()`, as used below. But I think we want to rethink Filter sooner rather than later, as the behavior of `getName()` with aliases is very confusing to me.